consensus/src/main/java/org/apache/iotdb/consensus/multileader/MultiLeaderServerImpl.java

```diff
@@ -479,6 +481,45 @@ public void notifyPeersToRemoveSyncLogChannel(Peer targetPeer)
     }
   }
 
+  public void waitTargetPeerUntilSyncLogCompleted(Peer targetPeer)
+      throws ConsensusGroupAddPeerException {
+    long checkIntervalInMs = 10_000L;
```

Comment:
Maybe this value can be a property of MultiLeaderConfig

Reply:
Both is fine. This parameter is used to avoid frequent useless queries and a fixed suitable interval is enough. For example, `10s` and `20s` are both ok and it is unnecessary to change it. So I think we can use a fixed value here